config.m4

```diff
@@ -52,9 +37,8 @@ if test "$PHP_SOLR" != "no"; then
         AC_MSG_ERROR([Solr extension requires json or jsonc support])
     fi
 
-    dnl until PHP 7.3: xml2-config or pkg-config
-    dnl since PHP 7.4: pkg-config only
-	PHP_SETUP_LIBXML(SOLR_SHARED_LIBADD, [
+	PHP_SETUP_LIBXML(SOLR_SHARED_LIBADD)
+
```

Comment:
Having everything in the "found" action is confusing
(if not found, error is raise by default)